outputs.tf

```
@@ -27,3 +27,8 @@ output "fqdn" {
   value       = join("", aws_route53_zone.default.*.name)
   description = "Fully-qualified domain name"
 }
+
+output "type" {
```

Can we make this output a boolean instead?
```suggestion
output "private_zone" {
```